strr-web/components/bcros/form-section/contact-information/Form.vue

```diff
@@ -200,16 +234,69 @@ onMounted(() => {
   }
 })
 
-const form = ref()
+const getActiveAddressState = (): ContactInformationI | CreateAccountFormStateI['propertyDetails'] => {
+  if (activeAddressField.value === 'primaryContactAddress') {
+    return formState.primaryContact
+  } else if (activeAddressField.value === 'secondaryContactAddress') {
+    return formState.secondaryContact
+  } else {
+    return formState.propertyDetails
+  }
+}
+
+// reset errors for Primary Contact
```

Comment:
Validation below needs to be reworked globally, but for now kept it for visibility